clients/scheduler/src/components/confirmation/index.tsx

```diff
@@ -38,9 +40,19 @@ export const Confirmation = () => {
     fetchScheduledAppointment(setAppointmentId)
   }, [fetchScheduledAppointment])
 
+  useEffect(() => {
+    if (!initialized) {
+      setInitialized(true)
```

Comment:
the setInitialized call(s) feel off to me, but we have it in several other places on the app so I'm assuming this is following a pattern that's currently working so no worries. should put refactor note somewhere for later